apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/ChatModel.kt

```diff
@@ -2335,6 +2335,10 @@ sealed class CIDirection {
     is LocalSnd -> true
     is LocalRcv -> false
   }
+
+  fun sameDirection(dir: CIDirection): Boolean {
```

Comment:
incorrectly call this function as sameDirection because it's name suggests to think about same as snd == snd, or DirectSnd == GroupSnd. Unclear the meaning, you have to read the code to understand. Since you use it only once, make this function local, or better just use it as an oneliner, like `val sameMemberAndDirection = ...`